scripts/printReleaseNotesTemplate.ts

```diff
@@ -1,55 +1,447 @@
+  const isFromV3 = revParsed === 'v3' || revParsed === 'v3-current'
+  const BASE_BRANCH = flags.to || (isFromV3 ? revParsed : 'next')
+  const PREV_RELEASE =
+    flags.from || execa.sync('git describe --abbrev=0', {shell: true}).stdout.trim()
 
-# Install or upgrade Sanity Studio
+  // Get current version
+  const packageJson = require('../package.json')
+  const currentVersion = packageJson.version
 
-To initiate a new Sanity Studio project or upgrade an existing one, please refer to our comprehensive guide on [Installing and Upgrading Sanity Studio](https://www.sanity.io/docs/upgrade).
+  // Generate changelog
+  const CHANGELOG_COMMAND = `git log --pretty=format:'%aN | %s | %h' --abbrev-commit --reverse ${PREV_RELEASE}..origin/${BASE_BRANCH}`
+  const changelogOutput = execa.sync(CHANGELOG_COMMAND, {shell: true}).stdout
 
-If you are updating from a version earlier than [3.37.0](https://www.sanity.io/changelog/93306939-973b-4e48-bb63-89a6b91d70b3), you should also check out [this article](https://www.sanity.io/help/upgrade-packages) to ensure your dependencies are up to date.
+  // Extract feature and bugfix commits for highlights
+  const commits = changelogOutput.split('\n').filter((line: string) => line.trim() !== '')
+  const features = commits.filter((commit: string) => {
+    const message = commit.split(' | ')[1].toLowerCase()
+    return message.includes('feat') || message.includes('feature') || message.includes('add')
+  })
+  const bugfixes = commits.filter((commit: string) => {
+    const message = commit.split(' | ')[1].toLowerCase()
+    return message.includes('fix') || message.includes('bug') || message.includes('issue')
+  })
 
-# 📓 Full changelog
-Author | Message | Commit
------------- | ------------- | -------------
-${execa.commandSync(CHANGELOG_COMMAND, {shell: true}).stdout}
-`
+  // Create release context
+  const releaseContext: ReleaseContext = {
+    version: currentVersion,
+    title: flags.title || `Sanity Studio v${currentVersion}`,
+    product: flags.product || 'Sanity Studio',
+    features,
+    bugfixes,
+    changelogOutput,
+  }
+
+  // Create Sanity documents if not in dry run mode
+  let changelogUrl = ''
+  let changelogId = ''
+  if (!flags.dryRun) {
+    // Get auth token from CLI config or from flag
+    const token = flags.token || getSanityAuthToken()
+
+    if (!token) {
+      console.error(
+        'Error: No auth token found. Please run `sanity login` or provide a token with --token',
+      )
+      process.exit(1)
+    }
+
+    const sanityConfig = {
+      projectId: PROJECT_ID,
+      dataset: DATASET,
+      token,
+      useCdn: false,
+      apiVersion: '2023-03-15',
+    }
+
+    try {
+      changelogId = await createSanityChangelogDocuments(sanityConfig, releaseContext, flags.debug)
+      // Internal Studio URL for editing (shown in console)
+      const studioUrl = `https://admin.sanity.io/structure/docs;changelog;apiChange;${changelogId}?perspective=rOVi7MsdW`
```

Comment:
This won't work after the `rOVi7MsdW` release is published, should it make a new release?